internal/backend/local/backend.go

```diff
@@ -447,7 +447,7 @@ func (b *Local) StatePaths(name string) (stateIn, stateOut, backupOut string) {
 // in the same files as the "new" state snapshots.
 func (b *Local) PathsConflictWith(other *Local) bool {
 	otherPaths := map[string]struct{}{}
-	otherWorkspaces, err := other.Workspaces()
+	otherWorkspaces, err := other.Workspaces(context.TODO())
```

Comment:
I'm curious about why we are adding context.TODO() here?  I only see this used by meta_backend.go which would be easy to stub out with a TODO or wire in more directly if possible.

Reply:
I was intentionally changing only the functions that directly call or directly implement the interface because I think anything more substantial than that ought to be in a separate PR that we can review without all of the distracting mechanical updates getting in the way. I am expecting that we make follow-up PRs to bridge these remaining gaps once the interfaces are all updated.